content/news/playtest-20151031.md

```diff
@@ -0,0 +1,173 @@
+</p>
+
+<hr />
+### 6. Joystick scrolling
+
+<div style="text-align:center" markdown="1">
+![Joystick scrolling](/images/news/20151031-scrolling.png)
+</div>
+
+<p>
+Another commonly requested feature was the ability to scroll using the right mouse button like a
+joystick (like TS and RA2).  You can now enable "Joystick" scrolling in the Settings input
+settings.  Yes, we are still working on our Tiberian Sun mod, and no we don't have a release date
+for you yet!
+</p>
+
+<hr />
+### 7. Units "defend" by default
+
+<div style="text-align:center" markdown="1">
+![RA Attack force](/images/news/20151031-radefend.png)
+</div>
+
+
+<p>
+Players have often complained about their units chasing the enemy across the map, and our "pro"
+players know that it is best to change the stance of their units to the "defend" mode.  We have
+made this the new default for all units, streamlining the game play a little bit for everyone.
+</p>
+
+<p>
+If you want to change your units back to the old "attack anything" mode you can use the change
+stance hotkey (ctrl+z by default).
+</p>
+
+<hr />
+### 8. New asset installer
+
+<div style="text-align:center" markdown="1">
+![Asset installer](/images/news/20151031-installer.png)
+</div>
+
+<p>
+We have consolidated our asset installation procedure into the mod chooser.  Look out for even more
+improvements in future releases!
+</p>
+
+<hr />
+### 9. Fixes and performance
+
+<div style="text-align:center" markdown="1">
+![Goodbye crashes!](/images/news/20151031-fixes.png)
+</div>
+
+<p>
+We have fixed a large number of bugs, including the well-known replay freeze bug when a player
+disconnects, and a memory corruption bug that would cause random crashes when starting a match on a
+Windows OS.  We have continued our crusade for performance, and this playtest shows significant
+further improvements over the last release.
+
+<!-- div style="border-radius: 4px; background-color: #272d2c; padding:10px; margin:10px;">
+<em>
+23:21 <+pchote> RoosterDragon: do you have numbers for how much better this playtest should be?<br />
+23:21 < RoosterDragon> a metric buttload<br />
+23:21 < RoosterDragon> you can quote me on that
+</em><br /><br />
+&mdash; from the openra IRC channel
+</div -->
+
+</p>
+
+<hr />
+### 10. Improved map and mod support
+
+<div style="text-align:center" markdown="1">
+![Mod support](/images/news/20151031-modders.png)
+</div>
+
+<p>
+There is a boatload of new features for modders and map authors: maps can now include custom music
+and define particle effects for rain, snow, or sand storms.  We have made major changes to the
+traits that control actor rendering and targeting that make them more flexible and self-consistent.
+Make sure to use the automatic rule upgrader to easily port your maps and mods to the new release!
```

Comment:
Maybe mention here that the UI changes will need manual adjustments if a mod has a custom UI and doesn't re-use an existing one.